cli/src/CliArgsParser.h

```diff
@@ -15,90 +15,26 @@
 * limitations under the License.
 ********************************************************************************/
 
-#ifndef OPENSPACENET_OPENSPACENETARGS_H
-#define OPENSPACENET_OPENSPACENETARGS_H
+#ifndef OPENSPACENET_CLIARGSPARSER_H
+#define OPENSPACENET_CLIARGSPARSER_H
 
+#include <OpenSpaceNetArgs.h>
+#include <OpenSpaceNet.h>
 #include <boost/program_options.hpp>
-#include <utility/Logging.h>
-#include <vector/Feature.h>
-
-#define OSN_LOG(sev) DG_LOG(OpenSpaceNet, sev)
-#define MAPSAPI_MAPID  "digitalglobe.nal0g75k"
-
 
 namespace dg { namespace osn {
 
-enum class Source
-{
-    UNKNOWN,
-    LOCAL,
-    DGCS,
-    EVWHS,
-    MAPS_API,
-    TILE_JSON
-};
-
-enum class Action
-{
-    UNKNOWN,
-    HELP,
-    DETECT,
-    LANDCOVER
-};
-
-class OpenSpaceNetArgs
+class CliArgsParser
```

Comment:
This is called `CliArgsParser`.  I would expect that to _only_ parse args (and maybe print errors/help during parsing). It shouldn't setup logging or have anything to do with processing.

Some solutions:
  1. Rename it something like `CommandLineOsn` (yeah, that name's terrible)
  2. Move the logging and actually starting processing into main.cpp
  3. Add another class that uses the `CliArgParser` and does the processing